### PR TITLE
code style: rename vars, use a function to compute LevelDB keys

### PR DESCRIPTION
**Pull request by derhuerst**
Will submit another PR that changes to tabs everywhere. Or do you prefer a different indentation style that I should apply?

### CODE REVIEW

**Comment by derhuerst** [comment]
Also, I would like to move the database opening logic out of `import` and instead pass it in as a parameter. What do you think?

File: read-gtfs/mapping.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	stop_times: st => 'stop_time-' + randomId() + '-' + st.trip_id,
+	calendar: s => 'service-' + s.service_id,
+	calendar_dates: e => 'calendar_date-' + randomId() + '-' + e.service_id
+}
+
+const dbToData = {
+	agency: agencyId => 'agency-' + agencyId,
+	stops: stopId => 'stop-' + stopId,
+	routes: routeId => 'route-' + routeId,
+	trips: (routeId, tripId) => 'trip-' + routeId + '-' + tripId,
```
Comment (juliuste) [inline read-gtfs/mapping.js]: This won't work if `routeID` is empty.

File: read-gtfs/mapping.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	agency: a => 'agency-' + a.agency_id,
+	stops: s => 'stop-' + s.stop_id,
+	routes: r => 'route-' + r.route_id,
+	trips: t => 'trip-' + t.route_id + '-' + t.trip_id,
+	stop_times: st => 'stop_time-' + randomId() + '-' + st.trip_id,
+	calendar: s => 'service-' + s.service_id,
+	calendar_dates: e => 'calendar_date-' + randomId() + '-' + e.service_id
+}
+
+const dbToData = {
```
Comment (juliuste) [inline read-gtfs/mapping.js]: Why does this exist? Does some other code use this?

Comment (derhuerst) [reply read-gtfs/mapping.js]: I should have committed in a more fine-grained fashion. It is the preparation for another PR I'm going to submit.

File: read-gtfs/mapping.js (hunk 1 above)
Comment (derhuerst) [reply read-gtfs/mapping.js]: When will `routeId` be empty?

Comment (juliuste) [reply read-gtfs/mapping.js]: When all trips are requested. In the legacy `tripStream` method for example.

Comment (derhuerst) [reply read-gtfs/mapping.js]: Ah. I changed this mapping to work with ranges of e.g. trips in my upcoming PR.

Let me know if you want these lines out of this PR, then I will move them to the other one.

Comment (juliuste) [reply read-gtfs/mapping.js]: I'd prefer to either have all changes in one PR or two strictly separated ones.

Comment (juliuste) [reply read-gtfs/mapping.js]: Therefore yes, or add some commits to this PR, I don't care 😄

**Pull request merged**